tiny_dnn/core/kernels/conv2d_op_avx.h

```diff
@@ -21,12 +21,12 @@ namespace kernels {
 #ifdef CNN_USE_AVX
 
 // float ver
-template <typename Allocator>
+template <typename S1, typename S2, typename S3, typename S4>
```

Comment:
Is it so in `Conv2dOp::compute` method that calls `kernels::conv2d_op_avx` function?
Will `core::OpKernelContext` class have different types (`view` and `tensor`) of in and out data and grad?